cypress/e2e/po/components/ResourceList/resource-list-masthead.po.ts

```diff
@@ -12,4 +12,8 @@ export default class ResourceListMastheadPo extends ComponentPo {
   create() {
     return this.self().find('[data-testid="masthead-create-yaml"]').click();
   }
+
+  createProject() {
```

Comment:
this is a generic component and should be resource agnostic

Reply:
Agreed. Upon a quick inspection, it looks like the existing `create()` is not used in any of our tests, so it should be safe to rename to `createYaml()`. Doing so should allow us to more accurately match the method to the test id.